src/Client.ts

```diff
@@ -925,4 +955,31 @@ export class Client extends EventEmitter<Events> {
 
     return new Message(this, message);
   }
+
+  async awaitMessage(
```

Comment:
could you rename to awaitMessage**s**?
also add methods to `DMChannel`, `GroupDMChannel` & `TextBasedGuildChannel` with the same name which call this.